src/plugins/dataflow/nodes/dataflow-node.tsx

```diff
@@ -27,7 +28,9 @@ export class DataflowNode extends Node {
 
     const dynamicClasses = classNames({
       "gate-active": node.data.gateActive,
-      "has-flow-in": hasFlowIn(node)
+      "has-flow-in": hasFlowIn(node),
```

Comment:
Technically we are not using these last two classes any more, but it can be useful to have classes like this in place, unless we think that is silly.